cmd/convox/apps.go

```diff
@@ -156,6 +156,7 @@ func cmdAppCreate(c *cli.Context) error {
 
 		if err := waitForAppRunning(c, app); err != nil {
 			stdcli.Error(err)
+			return err
```

Comment:
This could be changed to `return stdcli.Error(err)` to keep the context of a stdcli Error.